### PR TITLE
Update to use fs-nextra

### PR DESCRIPTION
**Pull request by kyranet**
### Proposed Semver Increment Bump: [MAJOR/MINOR/PATCH]
minor

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

- Use fs-nextra instead of fs-extra.

### (If Applicable) What Issue does it fix?
Nothing but performance, native promises ❤️

### CODE REVIEW

**Comment by kyranet** [comment]
With the PR https://github.com/bdistin/fs-nextra/pull/1, it seems to be working perfectly. Loading times decreased from `176.30ms` to `154.83ms` in my test.

**Comment by CyberiumShadow** [comment]
You speed whore

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "chalk": "^1.1.3",
     "discord.js": "hydrabolt/discord.js",
-    "fs-extra": "^3.0.0",
+    "fs-nextra": "0.0.1",
```
Comment (bdistin) [inline package.json]: "fs-nextra": "0.0.2",

Comment (kyranet) [reply package.json]: Done 👍

**Review by CyberiumShadow — APPROVED** [review]
lgtm

**Pull request merged**